next-docs/pages/world/introduction.mdx

```diff
@@ -1 +1,20 @@
 # Introduction
+
+[`Store`](/store/introduction) provides low-level primitives for storage.
+`World` provides the primitives for logic and access control on top of that storage.
+
+Calls are sent to the `World` as a central entry point.
+The `World` performs access control checks and if the request is authorized, sends it to the appropriate [`System`](/world/systems).
+Any access from the `System` to [the tables that store the data](/world/tables) also goes through the `World`, making it impossible to bypass access control.
```

Comment:
There is one exception to this: root systems, which bypass access control by using the internal libraries. Do you think it's worth mentioning that here?